packages/driver/cypress/integration/cypress/currentTest_spec.js

```diff
@@ -0,0 +1,53 @@
+let global_err = null
+
+try {
+  Cypress.currentTest
+} catch (e) {
+  global_err = e
+}
+
+describe('currentTest', () => {
+  let suite_err = null
+
+  try {
+    Cypress.currentTest
+  } catch (e) {
+    suite_err = e
+  }
+
+  before(() => {
+    expect(Cypress.currentTest.title).eq(cy.state('runnable').ctx.currentTest.title)
```

Comment:
This will pass even if both are `undefined`.

```suggestion
	expect(Cypress.currentTest.title).to.be.a('string')
    expect(Cypress.currentTest.title).eq(cy.state('runnable').ctx.currentTest.title)
```

... and move this to a function and do the same in the other hooks?